docs/UserGuide.md

```diff
@@ -1,12 +1,5 @@
----
-  layout: default.md
-  title: "User Guide"
-  pageNav: 3
----
+# WedLog User Guide
```

Comment:
Don't delete the first few lines shown below. These lines are headers, for example we need the `title` to set the HTML title of the page.
```
---
  layout: default.md
  title: "User Guide"
  pageNav: 3
---
```